moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -176,9 +177,27 @@ export const useTableComponents = ({
           orderDirection={orderByDirection}
         />
       ),
+      Toolbar: (props) => (
+        <Box
+          sx={{
+            display: "flex",
+            justifyContent: "space-between",
+          }}
+        >
+          <Box sx={{ mt: 2, ml: 2 }}>
+            {/* If there is previous data but no current data (i.e., useQuery
+              is refetching the data, then display a progress spinner) */}
+            {!!data && !projectListViewData && <CircularProgress />}
```

Comment:
iirc we used to have a bug where `loading` that is returned from Apollo `useQuery` used to never switch back to false, but that doesn't seem to be the case anymore. Maybe when we upgraded versions a while back, but, however it happened, it is nice!

How do you feel about using [Apollo's loading state](https://www.apollographql.com/docs/react/api/react/hooks/#loading) here instead?

Reply:
works perfectly and much smoother, thanks for the suggestion Mike!